external/gsEigen.cmake

```diff
@@ -0,0 +1,48 @@
+## Version: $Id:  $
+##
+##
+
+## Commentary:
+##
+##
+
+## Changelog:
+##
+##
+
+##
+## Code starts here
+## #############################################################################
+
+find_package(Eigen3 3.3 NO_MODULE)
+
+if(NOT Eigen3_FOUND)
+
+    message(STATUS "Eigen3 not found, fetching ...")
+
+    include(FetchContent)
```

Comment:
With the additions of FetchContent the minimum version of Cmake required should be revised to a higher number